cloud/kubernetes/helm/templates/_helpers.tpl

```diff
@@ -29,11 +29,9 @@ volumeMounts:
   mountPath: "/opt/tibco/be/{{ .Values.beShortVersion }}/rms/shared"
 {{- end }}
 {{- if .Values.rmsDeployment }}
-{{- if .Values.persistence.rmsSecurity }}
```

Comment:
We can delete this if condition for security and content to webstudion

Reply:
Yes this what the commit does.